src/book/02-system/01-foundations/03-continuous-integration.mdx

```diff
@@ -135,3 +157,35 @@ For more information about agent configuration and hooks, see the following page
 
 - <https://buildkite.com/docs/agent/v3/hooks>
 - <https://buildkite.com/docs/agent/v3/configuration>
+
+### Setup Pacman Package Cache
+
+<Alert type="info">
+
+Whenever we update our base archlinux image [here](https://github.com/NUbots/NUbots/blob/master/docker/Dockerfile#L13), both the mirror in the Dockerfile AND the mirror in the cache config need to be updated ([here](https://github.com/NUbots/NUbots/blob/master/docker/Dockerfile#L14) and [here](https://github.com/NUbots/NUbots/blob/master/doc/PacmanCache/docker-compose.yml#L14))
```

Comment:
Try to avoid links where the text is just "here".

```suggestion
Whenever we update our base Arch Linux image [here in the Dockerfile](https://github.com/NUbots/NUbots/blob/5149f1c0cd1e4ff902d749269d1aa90f52adc075/docker/Dockerfile#L13), both [the mirror in the Dockerfile](https://github.com/NUbots/NUbots/blob/5149f1c0cd1e4ff902d749269d1aa90f52adc075/docker/Dockerfile#L14) **_and_** [the mirror in the cache config](https://github.com/NUbots/NUbots/blob/master/doc/PacmanCache/docker-compose.yml#L14) need to be updated.
```